Doc/library/http.server.rst

```diff
@@ -362,7 +362,7 @@ instantiation, of which this module provides three different variants:
          delays, it now always returns the IP address.
 
 
-.. class:: SimpleHTTPRequestHandler(request, client_address, server, directory=None)
+.. class:: SimpleHTTPRequestHandler(request, client_address, server, directory=None, response_headers=None)
```

Comment:
Document that `directory` and `response_headers` are keyword arguments actually.

Reply:
Done in c376a7119c3a34192939801cc4f6409ada3f65a9